src/object/obj_rpc.h

```diff
@@ -164,8 +164,8 @@ enum obj_rpc_flags {
 
 /* common for update/fetch */
 #define DAOS_ISEQ_OBJ_RW	/* input fields */		 \
-	((struct dtx_id)	(orw_dti)		CRT_VAR) \
-	((daos_unit_oid_t)	(orw_oid)		CRT_VAR) \
+	((struct dtx_id)	(orw_dti)		CRT_RAW) \
+	((daos_unit_oid_t)	(orw_oid)		CRT_RAW) \
```

Comment:
Any changes here are a protocol change, and we should really be bumping the protocol number to force incompatibility.

Reply:
No, the protocol doesn't changed. This just describe how to serialize those members. It's used memcpy instead of custom user function with CRT_RAW flag.

Reply:
Ah, that's good to know.